tests/models/idefics/test_modeling_idefics.py

```diff
@@ -268,10 +293,34 @@ def setUp(self):
     def test_config(self):
         self.config_tester.run_common_tests()
 
-    def test_model(self):
-        config_and_inputs = self.model_tester.prepare_config_and_inputs()
+    def test_model_single_image(self):
+        config_and_inputs = self.model_tester.prepare_config_and_inputs(
+            num_images=1, interpolate_pos_encoding=False, image_expansion=0
+        )
+        self.model_tester.create_and_check_model(*config_and_inputs)
+
+    def test_model_multiple_images(self):
+        config_and_inputs = self.model_tester.prepare_config_and_inputs(
+            num_images=2, interpolate_pos_encoding=False, image_expansion=0
+        )
         self.model_tester.create_and_check_model(*config_and_inputs)
 
+    def test_model_with_image_pos_embeddings_interpolation(self):
```

Comment:
interpolate is never tested with 2 images is this expected to not work?

Reply:
No, the opposite. I added the multiple image tests for interpolation and generation